### PR TITLE
implemented easily switching to previous tag

### PR DESCRIPTION
**Pull request by dakyskye**
#51

### CODE REVIEW

**Comment by 0xACE** [comment]
This PR contains a lot of style fixes in the code which are not relevant to the commit itself.

It's better that you re-make your PR to only handle the issue at hand.

And maybe in the future the project should avoid having tailing spaces and format/coding style issues.

@avahe-kellenberger what do you think about adding a `nimpretty` check for this project?

**Comment by dakyskye** [comment]
Yeah, my Vim is set to use tabs, not spaces, and I just grabbed one of the my .editorconfig files for it, which apparently fixed styling issues.

**Comment by avahe-kellenberger** [comment]
I'll configure nimpretty after this PR, thanks for the tip

File: src/nimdowpkg/windowmanager.nim [hunk 1]
```diff
@@ -343,9 +343,21 @@ proc mapConfigActions*(this: WindowManager) =
     this.focusNextMonitor()
 
   createControl(keycode, "goToTag"):
-    let tag = this.selectedMonitor.keycodeToTag(keycode)
+    var tag = this.selectedMonitor.keycodeToTag(keycode)
+    if this.selectedMonitor.previousTag != -1 and this.selectedMonitor.selectedTag.id == tag.id:
+      let currentTag = tag.id
+      tag = this.selectedMonitor.keycodeToTag(this.selectedMonitor.previousTag+10)
```
Comment (avahe-kellenberger) [inline src/nimdowpkg/windowmanager.nim:349]: We can avoid doing this if we use an actual tag instead of an int - Tags are ref objects so it's fine

Comment (dakyskye) [reply src/nimdowpkg/windowmanager.nim:349]: I tried just modifying id key for tag object, without +10, but it did not work.

Comment (avahe-kellenberger) [reply src/nimdowpkg/windowmanager.nim:349]: I mean using the type Tag instead of an int. You wouldn't need to modify the id, that shouldn't change

Comment (dakyskye) [reply src/nimdowpkg/windowmanager.nim:349]: Oh, good point.  I will rewrite it tomorrow :+1:

**Comment by dakyskye** [comment]
It took me a couple of force pushes to revert cleanup!

**Pull request merged**